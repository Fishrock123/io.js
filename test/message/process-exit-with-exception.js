'use strict';

const common = require('../common');
common.disableCrashOnUnhandledRejection();

process.on('unhandledRejection', (err) => {
  process.exitWithException(err);
});

Promise.reject(new Error('uh oh'));

setImmediate(common.mustNotCall);
