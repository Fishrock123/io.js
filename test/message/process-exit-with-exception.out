*test*message*process-exit-with-exception.js:*
Promise.reject(new Error('uh oh'));
               ^

Error: uh oh
    at Object.<anonymous> (*test*message*process-exit-with-exception.js:*:*)
    at *
    at *
    at *
    at *
    at *
    at *
